### PR TITLE
OCPBUGS-20183: PTP4L output in T-GM 

### PR DESCRIPTION
**Pull request by aneeshkp**
set time property to true for clock class 6 and 7

### CODE REVIEW

File: pkg/event/event_test.go [hunk 1]
```diff
@@ -35,8 +35,8 @@ func monkeyPatch() {
 				CurrentUtcOffsetValid: false,
```
Comment (jnunyez) [inline pkg/event/event_test.go:35]: Is this the equivalent to UTC_REASONABLE?

File: pkg/event/event_test.go [hunk 2]
```diff
@@ -35,8 +35,8 @@ func monkeyPatch() {
...
 				Leap59:                false,
 				Leap61:                false,
-				TimeTraceable:         false,
-				FrequencyTraceable:    false,
+				TimeTraceable:         true,
+				FrequencyTraceable:    true,
 				PtpTimescale:          false,
 				TimeSource:            0,
```
Comment (jnunyez) [inline pkg/event/event_test.go:41]: Is this the equivalent to TimeSource in ptp4l packets? Should 0x20 (GPS) for T-GM when GNSS connected.

Comment (aneeshkp) [reply pkg/event/event_test.go:41]: updated to GNSS when connected . and set to unknown when GNSS is lost

File: pkg/event/event_test.go (hunk 1 above)
Comment (aneeshkp) [reply pkg/event/event_test.go:35]: set CurrentUtcOffsetValid  to true for 6 and 7

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**